libsql_client/hrana/conn.py

```diff
@@ -250,8 +250,9 @@ def _close_stream(self, stream_state: _StreamState, e: BaseException) -> None:
             return
         stream_state.closed = e
 
-        def close_done(_fut: asyncio.Future[proto.Response]) -> None:
+        def close_done(fut: asyncio.Future[proto.Response]) -> None:
             self._stream_id_alloc.free(stream_state.stream_id)
+            fut.exception()  # check it so it won't be reported
```

Comment:
The future may be cancelled, in which case the `exception()` method would throw an exception. `if not fut.cancelled(): fut.exception()` will fix that.

Reply:
well noticed

Reply:
done!